src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -337,4 +355,15 @@ def cleanse(self, target_path, arguments, input_path, output_path, max_time):
     Returns:
       A boolean indicating success.
     """
-    raise NotImplementedError
+    runner = libfuzzer.get_runner(target_path)
+    launcher.set_sanitizer_options(target_path)
+
+    cleanse_tmp_dir = self._create_temp_corpus_dir('cleanse-workdir')
+    artifact_prefix = constants.ARTIFACT_PREFIX_FLAG + cleanse_tmp_dir + '/'
```

Comment:
nit: can create a helper for _get_artifact_prefix(output_dir)

Reply:
Done!